app/helpers/dossier_helper.rb

```diff
@@ -78,6 +78,17 @@ def class_badge_state(state)
     end
   end
 
+  def status_badge_user(dossier, alignment_class = '')
+    if dossier.hide_info_with_accuse_lecture?
+      tag.span 'traité', role: 'status', class: class_names(
+        'fr-badge fr-badge--sm fr-badge--no-icon' => true,
+        alignment_class => true
+      )
```

Comment:
tu peux pas faire directe 

```suggestion
      tag.span 'traité', role: 'status', class: 'fr-badge.fr-badge--sm.fr-badge--no-icon.alignment_class'
```

?